contrib/ci/Jenkinsfile

```diff
@@ -1,27 +1,46 @@
+def lualint() {
+	make lint-lua
+}
+
+def shelllint() {
+	make lint-sh
+}
+
+run_lint_set = [
+	"task1": {
+		lualint()
+	},
+	"task2": {
+		shelllint()
+	}
+]
+
 pipeline {
-    agent { label 'gluon-docker' }
-    environment {
-        GLUON_SITEDIR = "contrib/ci/minimal-site"
-        GLUON_TARGET = "x86-64"
-        BUILD_LOG = "1"
-    }
-    stages {
-        stage('lint') {
-            steps {
-                sh 'luacheck package scripts targets'
-            }
-        }
-        stage('docs') {
-            steps {
-                sh 'make -C docs html'
-            }
-        }
-        stage('build') {
-            steps {
-                sh 'make update'
-                sh 'test -d /dl_cache && ln -s /dl_cache openwrt/dl || true'
-                sh 'make -j$(nproc) V=s'
-            }
-        }
-    }
+	agent { label 'gluon-docker' }
+	environment {
+		GLUON_SITEDIR = "contrib/ci/minimal-site"
+			GLUON_TARGET = "x86-64"
+			BUILD_LOG = "1"
+	}
+	stages {
+		stage('lint') {
+			steps {
+				script {
+					parallel(run_lint_set)
+				}
+			}
+		}
+		stage('docs') {
+			steps {
+				sh 'make -C docs html'
+			}
+		}
+		stage('build') {
+			steps {
+				sh 'make update'
+					sh 'test -d /dl_cache && ln -s /dl_cache openwrt/dl || true'
+					sh 'make -j$(nproc) V=s'
+			}
+		}
+	}
```

Comment:
While the indentation fixes are welcome, they make the review harder than necessary. You can do both in the same PR, but only if you squash all your fixups into meaningful commits, for example:

1. Addition of lint scripts and make targets
2. Jenkinsfile indentation fix
3. Jenkinsfile lint implementation